core/embed/rust/src/trezorhal/display.rs

```diff
@@ -5,40 +5,19 @@ use core::ptr;
 
 pub use ffi::{DISPLAY_RESX, DISPLAY_RESY};
 
+pub type FontInfo = ffi::font_info_t;
+
 pub fn backlight(val: i32) -> i32 {
     unsafe { ffi::display_set_backlight(val) }
 }
 
-pub fn text_width(text: &str, font: i32) -> i16 {
+pub fn get_font_info(font: i32) -> Option<FontInfo> {
     unsafe {
-        ffi::font_text_width(font, text.as_ptr() as _, text.len() as _)
-            .try_into()
-            .unwrap_or(i16::MAX)
+        let font = ffi::get_font_info(font);
+        Some(*font.as_ref()?)
```

Comment:
this could use a SAFETY comment -- something along the lines of "ffi::get_font_info returns a pointer to ROM data which is 'static and immutable"

Reply:
dd36660f473954787c3d6929f4d122f3f2acfb9c